### DISCUSSION TITLE
langflow behind jupyter server proxy

### DISCUSSION THREAD

**Discussion by bijucyborg** [langflow-ai/langflow#658] [Q&A]
Hi,

Thanks for such a useful tool. I was trying to integrate langflow as an jupyter notebook extension. 

The problem is though that the webserver does not accept any parameter as a base path. Jupyter will attempt to serve the assets via http://JUPYTERURL/<extension_name> so in this case http://127.0.0.1:8848/langchain

Currently the index page tries to load the assets such --> http://127.0.0.1:8848/assets/index-baa2e6b2.js whereas its available at 
http://127.0.0.1:8848/langchain/assets/index-baa2e6b2.js

I tried setting SCRIPT_NAME as environment variable but for some reason Gunicorn does not consider it all. I don't wish to modify the code and hoping that this can be achieved either via environment variables or command line arguments.

**Comment by Chukarslan** [comment]
Utility-wise you're just better off using Python directly. I can see this becoming an issue if you were to use Sagemaker for eg. but for that you can just spool up an EC2 instance (or similar for other cloud providers). 

Personally feel like this is not a good way to use Langflow :)

**Comment by ogabrielluiz** [comment]
Hey @bijucyborg 

This is an interesting idea. Do you think you can point me to what exactly you would like Langflow to have? 
Maybe I can take a crack at adding that option for you.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
